packages/x-charts/src/internals/plugins/featurePlugins/useChartInteraction/useChartInteraction.types.ts

```diff
@@ -17,35 +19,22 @@ export interface UseChartInteractionInstance {
    */
   removeItemInteraction: (itemToRemove?: ChartItemIdentifier<ChartSeriesType>) => void;
   /**
-   * Set the new axis the user is interacting with.
-   * @param {Partial<AxisInteractionData>} newAxis The new axis identifier.
+   * Set the new pointer coordinate.
+   * @param {Coordinate | null} newCoordinate The new pointer coordinate.
    */
-  setAxisInteraction: (newAxis: Partial<AxisInteractionData>) => void;
+  setPointerCoordinate: (newCoordinate: Coordinate | null) => void;
 }
 
-export type AxisInteractionData = {
-  x: null | {
-    value: number | Date | string;
-    // Set to -1 if no index.
-    index: number;
-  };
-  y: null | {
-    value: number | Date | string;
-    // Set to -1 if no index.
-    index: number;
-  };
-};
-
 export interface UseChartInteractionState {
   interaction: {
     /**
      * The item currently interacting.
      */
     item: null | ChartItemIdentifier<ChartSeriesType>;
     /**
-     * The x- and y-axes currently interacting.
+     * The x/y SVG coordinate of the "main" pointer
```

Comment:
What is the "main" pointer?

Reply:
I thaught about multip pointer interaction. Since we only store one point, it's the main one. How to handle this multi-point is an open quesiton